supervisord.conf

```diff
@@ -66,7 +66,7 @@ stopasgroup = true
 [program:prometheus]
 priority = 7
 user = pmm
-command = /usr/sbin/prometheus -config.file=/etc/prometheus.yml -storage.local.path=/opt/prometheus/data -web.listen-address=:9090 -storage.local.retention=ENV_METRICS_RETENTION -storage.local.memory-chunks=ENV_METRICS_MEMORY -storage.local.chunk-encoding-version=2 -web.console.libraries=/usr/share/prometheus/console_libraries -web.console.templates=/usr/share/prometheus/consoles -web.external-url=http://localhost:9090/prometheus/
+command = /usr/sbin/prometheus -config.file=/etc/prometheus.yml -storage.local.path=/opt/prometheus/data -web.listen-address=:9090 -storage.local.retention=ENV_METRICS_RETENTION --storage.local.target-heap-size=ENV_METRICS_MEMORY_MULTIPLIED -storage.local.chunk-encoding-version=2 -web.console.libraries=/usr/share/prometheus/console_libraries -web.console.templates=/usr/share/prometheus/consoles -web.external-url=http://localhost:9090/prometheus/
```

Comment:
not always possible to split long lines in supervisord config file.
because it has quite interesting parser - https://github.com/Supervisor/supervisor/issues/140
anyway fixed for prometheus, the the next commit